client/app/bundles/course/assessment/pages/AssessmentStatistics/LiveFeedbackStatisticsTable.tsx

```diff
@@ -0,0 +1,306 @@
+        className: 'text-right',
+        sortProps: {
+          sort: (a, b): number => {
+            return sortNullableLiveFeedbackCount(
+              a.liveFeedbackCount?.[index] ?? null,
+              b.liveFeedbackCount?.[index] ?? null,
+            );
+          },
+        },
+      };
+    });
+
+  const columns: ColumnTemplate<AssessmentLiveFeedbackStatistics>[] = [
+    {
+      searchProps: {
+        getValue: (datum) => datum.courseUser.name,
+      },
+      title: t(translations.name),
+      sortable: true,
+      searchable: true,
+      cell: (datum) => (
+        <div className="flex grow items-center">
+          <Link to={`/courses/${courseId}/users/${datum.courseUser.id}`}>
+            {datum.courseUser.name}
+          </Link>
+          {datum.courseUser.isPhantom && (
+            <GhostIcon className="ml-2" fontSize="small" />
+          )}
+        </div>
+      ),
+      csvDownloadable: true,
+    },
+    {
+      of: 'groups',
+      title: t(translations.group),
+      sortable: true,
+      searchable: true,
+      searchProps: {
+        getValue: (datum) => getJointGroupsName(datum.groups),
+      },
+      cell: (datum) => getJointGroupsName(datum.groups),
+      csvDownloadable: true,
+    },
+    {
+      of: 'workflowState',
+      title: t(translations.workflowState),
+      sortable: true,
+      cell: (datum) => (
+        <Chip
+          className="w-100"
+          label={translateStatus(
+            datum.workflowState ?? workflowStates.Unstarted,
+          )}
+          style={{
+            backgroundColor:
+              palette.submissionStatus[
+                datum.workflowState ?? workflowStates.Unstarted
+              ],
+          }}
+          variant="filled"
+        />
+      ),
+      className: 'center',
+    },
+    ...statColumns,
+    {
+      searchProps: {
+        getValue: (datum) =>
+          datum.liveFeedbackCount
+            ? datum.liveFeedbackCount
+                .reduce((sum, count) => sum + (count || 0), 0)
+                .toString()
+            : '',
+      },
+      title: t(translations.totalFeedbackCount),
+      cell: (datum): ReactNode => {
+        const totalFeedbackCount = datum.liveFeedbackCount
+          ? datum.liveFeedbackCount.reduce(
+              (sum, count) => sum + (count || 0),
+              0,
+            )
+          : null;
+        return (
+          <div className="p-[1rem]">
+            <Box>{totalFeedbackCount}</Box>
+          </div>
+        );
+      },
+      sortable: true,
+      csvDownloadable: true,
+      className: 'text-right',
+      sortProps: {
+        sort: (a, b): number => {
+          const totalA = a.liveFeedbackCount
+            ? a.liveFeedbackCount.reduce((sum, count) => sum + (count || 0), 0)
+            : 0;
+          const totalB = b.liveFeedbackCount
+            ? b.liveFeedbackCount.reduce((sum, count) => sum + (count || 0), 0)
+            : 0;
+          return totalA - totalB;
```

Comment:
same issue of repeated computations. Also, why is this sort logic different from the other sort above?  
This one excludes name, phantom status

Reply:
The sort logic is different because this is for the sorting on a specific column (when clicking on table name)